base_location/models/res_partner.py

```diff
@@ -106,35 +106,35 @@ def _check_zip(self):
             error_dict = {"partner": rec.name, "location": rec.zip_id.name}
             if rec.zip_id.city_id.country_id != rec.country_id:
                 raise ValidationError(
-                    _(
+                    self.env._(
```

Comment:
AFAIK, there's no need to change this expression. Is there any reason to change it?

Reply:
not strictly required but it offers a minor performance improvement, see https://github.com/odoo/odoo/pull/174844

Reply:
Thanks for the pointer. Adding this to the migration guide.